content/applications/marketing/survey/time_random.rst

```diff
@@ -0,0 +1,77 @@
+============================
+Timed & randomized questions
+============================
```

Comment:
As for the scoring page, I will skip reviewing that page for now, given that you may change the way you organize your Surveys docs.